prebuilt_swift_static_framework/README.md

```diff
@@ -0,0 +1,27 @@
+# prebuilt_swift_static_framework
+
+This is a macro & rule to build multi-architecture (fat) static
+frameworks from Swift sources.
```

Comment:
If you're up for it, maybe a blurb pointing to the use of `cfg = apple_common.multi_arch_split` and `ctx.split_attr.deps`, since these are not sufficiently documented and are what makes this all work.

Reply:
sure

Reply:
Added some more docs, lmk what you think

Reply:
Looks good, thanks